cmake/developer_package.cmake

```diff
@@ -4,10 +4,27 @@
 
 cmake_minimum_required(VERSION 3.13)
 
+# Detect target
+include(target_flags)
+
+string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} ARCH_FOLDER)
+if(X86_64)
+    set(ARCH_FOLDER intel64)
+elseif(X86)
+    set(ARCH_FOLDER ia32)
+elseif(MSVC AND ARM)
+    set(ARCH_FOLDER arm)
+elseif(MSVC AND AARCH64)
+    set(ARCH_FOLDER arm64)
```

Comment:
arm64 is not an architecture. Maybe it make sense to align with Linux?

Reply:
I mean aarch64 for 64bit and others for 32bit

Reply:
In short words - I'd like to propose to introduce as less confusion as possible - single abbreviation if possible

Reply:
In MSVC we pass `-A ARM64`, so I have decided that `arm64` is more native for Windows path. Is not it?

Reply:
Why not to keep `string(TOLOWER ${CMAKE_SYSTEM_PROCESSOR} ARCH)` ? It seems more generic

Reply:
because for MSVC building for different architecture is not a cross-completion. So, the system processor is still x86.